src/sqlancer/feldera/FelderaSchema.java

```diff
@@ -1,11 +1,9 @@
 package sqlancer.feldera;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Objects;
+import java.util.*;
 import java.util.stream.Collectors;
 
+import com.yugabyte.jdbc.EscapeSyntaxCallMode;
```

Comment:
why is this needed?

Reply:
That is not needed, I didn't even notice that. 
Will remove.